vividus-plugin-applitools/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -68,4 +68,17 @@
         <idref bean="visualTestingSteps" />
         <idref bean="ufgSteps" />
     </util:list>
+
+    <beans profile="executioncloud">
+        <bean class="org.vividus.executioncloud.selenium.ExecutionCloudUrlProvider" primary="true" />
```

Comment:
isn't profile order enough to set the order of beans loading?

Reply:
no, why did you think so?

Reply:
1) VIVIDUS profiles order matters
2) If Spring loads profiles exactly in the same order they defined, then we should rely on bean loading order, rather then marking some beans primary

Reply:
vividus is vividus, spring is spring, so order of profiles in spring doesn't make early loaded beans primary over lately loaded